dino.icu.yaml

```diff
@@ -264,6 +264,11 @@ periodic:
    type: CNAME
    value: leecheeyong.github.io.
    
+piyush:
+   ttl: 1
```

Comment:
TTLs should be set to a minimum of 600 seconds (or ten minutes), to allow for proper DNS server caching

Reply:
My apologies.

That's what I had in mind, but the comments explicitly said to configure TTL as 1, and that's how all the other records are also configured as.

https://github.com/hackclub/dns/blob/467d3839e4d873f379b90b7c910ebd7c99abc90a/dino.icu.yaml#L228-L231

Changing straight away!

Reply:
Yes, this needs to be updated - my apologies about the docs #1023